contracts/staking/test/utils/cumulative_reward_tracking_simulation.ts

```diff
@@ -36,12 +36,7 @@ export class CumulativeRewardTrackingSimulation {
     private static _extractTestLogs(txReceiptLogs: DecodedLogs): TestLog[] {
         const logs = [];
         for (const log of txReceiptLogs) {
-            if (log.event === TestCumulativeRewardTrackingEvents.SetMostRecentCumulativeReward) {
```

Comment:
Why are we removing this?

Reply:
We inlined `_forceSetMostRecentBlahBlah()` so it's not hookable anymore.